doc/src/fix_wall_ACL.txt

```diff
@@ -0,0 +1,72 @@
+"LAMMPS WWW Site"_lws - "LAMMPS Documentation"_ld - "LAMMPS Commands"_lc :c
+
+:link(lws,http://lammps.sandia.gov)
+:link(ld,Manual.html)
+:link(lc,Section_commands.html#comm)
+
+:line
+
+fix wall/ACL command :h3

```

Comment:
please follow LAMMPS' naming conventions. that is all documented style names must be lowercase, should be descriptive (avoid acronyms unless they are very common) and separate multiple words with a forward slash '/'.
in file names (source code and documentation) the all lower-case should be used as well, and then words separated by underscores '_'.  C++ class names then use a CamelCase convention.